operators/admin/olm-cs-podsched.adoc

```diff
@@ -6,14 +6,21 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-When an Operator Lifecycle Manager (OLM) catalog source of source type `grpc` defines a `spec.image`, the Catalog Operator creates a pod that serves the defined image content. By default, this pod defines the following in its spec:
+When an Operator Lifecycle Manager (OLM) catalog source of source type `grpc` defines a `spec.image`, the Catalog Operator creates a pod that serves the defined image content. By default, this pod defines the following in its specification:
 
 * Only the `kubernetes.io/os=linux` node selector
 * No priority class name
 * No tolerations
 
 As an administrator, you can override these values by modifying fields in the `CatalogSource` object's optional `spec.grpcPodConfig` section.
 
+[NOTE]
+====
+The Marketplace Operator manages the default `OperatorHub` custom resource (CR), including the CR's catalog source pods. 
+
+If you want to override the settings for the default catalog source pods, you must remove `openshift-marketplace` from the `namespace` field in the `CatalogSource` object and then enter your custom namespace in this field. This configuration ensures that the `CatalogSource` object retains its original name and points to the original image tag. You can then modify the image tag to meet your requirements.
```

Comment:
Ah, no, sorry I wouldn't recommend that. The `openshift-marketplace` namespace is special in that catalogs installed in that namespace are available globally on the cluster. Also, if you modify those default catalogs that ship with openshift without touching the config api, the marketplace operator will just revert those changes. In my previous comment I was suggesting that you actually use the configuration API called `operatorhub` that was built specifically to solve this problem.

There's some documentation here that goes into how to use it, although it is in the context of disconnected clusters:

https://docs.openshift.com/container-platform/4.8/operators/admin/olm-restricted-networks.html#olm-restricted-networks-operatorhub_olm-restricted-networks

Reply:
Thanks, @kevinrizza . I updated the content based on your feedback. Should I go into details with command examples that show a user what they might be able to do?

Reply:
Does `OperatorHub` CR and `OperatorHub` object mean the same thing? 

See the following links:

* [OperatorHub custom resource](https://docs.openshift.com/container-platform/4.13/operators/understanding/olm-understanding-operatorhub.html)

* [Disabling the default OperatorHub catalog sources](https://docs.openshift.com/container-platform/4.13/operators/admin/olm-restricted-networks.html#olm-restricted-networks-operatorhub_olm-restricted-networks)

Reply:
yes, the custom resource is colloquially referred to as the instance of the object. they're more or less interchangeable

Reply:
I think it's reasonable to add reference to disabling the default as long as those links are there. with that, this lgtm